class-tgm-plugin-activation.php

```diff
@@ -2272,7 +2278,8 @@ protected function get_plugin_status_text( $slug ) {
 			}
 
 			return sprintf(
-				_x( '%1$s, %2$s', '%1$s = install status, %2$s = update status', 'tgmpa' ),
+				/* translators: %1$s: install status, %2$s: update status */
+				__( '%1$s, %2$s', 'tgmpa' ),
 				$install_status,
```

Comment:
As this 'string' is so generic, should it not have a translators comment *and* a context through _x ?